### DISCUSSION TITLE
Recovered Mass From Pumping wells

### DISCUSSION THREAD

**Discussion by mahaderony** [modflowpy/flopy#1478] [Q&A] [answered]
Hi, 
Is there any way to recover the mass out from a well after and MT3DMS simulation? I have a transport model and I want to know how much mass is exiting from each well during each stress period.

**Comment by emorway-usgs** [comment, marked as answer]
This is an MT3DMS question and has very little (if anything) to do with FloPy.  The MT3DMS manual is available for download [here](https://hydro.geo.ua.edu/mt3d/index.htm).  There's a helpful list of the output provided by MT3DMS on page 85:  

The Standard Output File contains echo of input data (so that they can be
checked to ensure they have been read in properly), printout of calculated
concentrations for each species, and some other useful information such
**as model-calculated** parameters and **mass budgets**, at user-specified times
and at the end of each stress period.

In your standard output listing file, you can search for the following string "CUMULATIVE MASS BUDGETS AT END OF TRANSPORT STEP" That should help you get started.
